try:
  num = int(input('Enter a number'))
except Exception as e:
  print(f'Error: {e}')
  